android/onnxruntime-native/build.gradle

```diff
@@ -0,0 +1,201 @@
+    def signingKey = findProperty("signingKey")
+    def signingPassword = findProperty("signingPassword")
+    useInMemoryPgpKeys(signingKey, signingPassword)
+    sign publishing.publications
+}
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                from components.release
+                pom {
+                    name = "ONNXRuntime Android native library"
+                    description = "ONNXRuntime Android native library binary distribution"
+                    url = "http://www.djl.ai/android"
+                    packaging = "aar"
+
+                    licenses {
+                        license {
+                            name = "The Apache License, Version 2.0"
+                            url = "https://www.apache.org/licenses/LICENSE-2.0"
+                        }
+                    }
+
+                    scm {
+                        connection = "scm:git:git@github.com:deepjavalibrary/djl.git"
+                        developerConnection = "scm:git:git@github.com:deepjavalibrary/djl.git"
+                        url = "https://github.com/deepjavalibrary/djl"
+                        tag = "HEAD"
+                    }
+
+                    developers {
+                        developer {
+                            name = "DJL.AI Team"
+                            email = "djl-dev@amazon.com"
+                            organization = "Amazon AI"
+                            organizationUrl = "https://amazon.com"
+                        }
+                    }
+                }
+            }
+        }
+
+        repositories {
+            maven {
+                if (project.hasProperty("snapshot")) {
+                    name = "snapshot"
+                    url = "https://oss.sonatype.org/content/repositories/snapshots/"
+                    credentials {
+                        username = findProperty("ossrhUsername")
+                        password = findProperty("ossrhPassword")
+                    }
+                } else if (project.hasProperty("staging")) {
+                    name = "staging"
+                    url = "https://oss.sonatype.org/service/local/staging/deploy/maven2/"
+                    credentials {
+                        username = findProperty("ossrhUsername")
+                        password = findProperty("ossrhPassword")
+                    }
+                } else {
+                    name = "local"
+                    url = "build/repo"
+                }
+            }
+        }
+    }
+}
+configurations {
+    exclusion
+}
+
+dependencies {
+//    api "ai.djl:api:${version}"
+//    api "ai.djl.android:core:${version}"
+    embed  "ai.djl.onnxruntime:onnxruntime-engine:0.18.0"
+    embed  "com.microsoft.onnxruntime:onnxruntime-android:1.11.0"
+    exclusion "com.microsoft.onnxruntime:onnxruntime"
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+//apply plugin: 'java-library'
```

Comment:
This is the setup for transient onnxruntime, @frankfliu what do you think?

Reply:
Please try my PR: https://github.com/deepjavalibrary/djl/pull/1866

Your android project should be able to simply include:
```
implementation "ai.djl.android:onnxruntime:0.19.0-SNAPSHOT"
```